examples/authentication.py

```diff
@@ -82,3 +82,49 @@ def token_gen_call(username, password):
     if mockpassword == password and mockusername == username: # This is an example. Don't do that.
         return {"token" : jwt.encode({'user': username, 'data': 'mydata'}, secret_key, algorithm='HS256')}
     return 'Invalid username and/or password for user: {0}'.format(username)
+
+# JWT AUTH EXAMPLE #
+replace_this = False
+config = {
+    'jwt_secret': 'super-secret-key-please-change',
+    # token will expire in 3600 seconds if it is not refreshed and the user will be required to log in again
```

Comment:
Well, I mean normally sentences have a Capital letter at the start and ends with a dot 😝. But I can see that this projects sentences don't end with a dot xD

Ex: https://github.com/timothycrosley/hug/blob/a2dfdd71dc5cb809107ca710fd6c0629c6802cfc/hug/api.py#L77

So lets stick with the project's standard and use a Capital letter for the begging of the sentence. 😉